pkg/volume/flexvolume/probe.go

```diff
@@ -173,7 +173,7 @@ func (prober *flexVolumeProber) handleWatchEvent(event fsnotify.Event) error {
 			if err := prober.createPluginDir(); err != nil {
 				return err
 			}
-			if err := prober.addWatchRecursive(pluginDirAbs); err != nil {
+			if err := prober.addWatchRecursive(prober.pluginDir); err != nil {
```

Comment:
not sure why we're changing this away from being the absolute path?

Reply:
Reverted.